src/test/librados/TestCase.cc

```diff
@@ -343,8 +344,8 @@ void RadosTestPP::cleanup_namespace(librados::IoCtx ioctx, std::string ns)
     ioctx.locator_set_key(it->get_locator());
     ObjectWriteOperation op;
     op.remove();
-    librados::AioCompletion *completion = s_cluster.aio_create_completion();
-    ASSERT_EQ(0, ioctx.aio_operate(it->get_oid(), completion, &op,
+    std::unique_ptr<librados::AioCompletion> completion(s_cluster.aio_create_completion());
```

Comment:
We don't get to the release call if we fail the ASSERT_EQ call. See Coverity issues 1395734 and 1395735. Is there a reason not to use smart pointers here?